mmv1/products/gkeonprem/VmwareCluster.yaml

```diff
@@ -100,7 +100,6 @@ properties:
       Prefix must be a DNS subdomain.
       Name must be 63 characters or less, begin and end with alphanumerics,
       with dashes (-), underscores (_), dots (.), and alphanumerics between.
-    default_from_api: true
```

Comment:
double checking / need a reminder - there is another field that is `computed` that will replace the functionality of this field being `default_from_api` right? And this field is moving to write-only?